### ISSUE TITLE
bug: Can't pull LocalStack docker image

### ISSUE THREAD

**Issue by IshakAtLEGO** [localstack/localstack#5384]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I am trying to run LocalStack in docker container but am getting an error :

```shell
$ docker compose -f docker-compose.localstack.yml up -d
[+] Running 0/1
 - localstack Error                                                                                                                                                                                                                                  2.2s 
Error response from daemon: pull access denied for localstack/localstack, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

my
I am getting same error with `docker run` or `docker pull`. 
I have no issues with other docker images.

regards

### Expected Behavior

loclastack docker container runs locally.

### How are you starting LocalStack?

With a docker-compose file

### Steps To Reproduce

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

    docker compose -f docker-compose.localstack.yml up -d

#### Docker compose file docker-compose.localstack.yml : 

```yml
version: "3.8"

services:
  localstack:
    container_name: "${LOCALSTACK_DOCKER_NAME-}"
    image: localstack/localstack
    network_mode: bridge
    ports:
      - "4566:4566"
      - "4571:4571"
    environment:
      - LOCALSTACK_DOCKER_NAME=${LOCALSTACK_DOCKER_NAME-}
      - HOSTNAME=localstack
      - HOSTNAME_EXTERNAL=localstack
      - DEFAULT_REGION=eu-west-1
      - SERVICES=${SERVICES- }
      - DEBUG=${DEBUG- }
      - DATA_DIR=${DATA_DIR- }
      - LAMBDA_EXECUTOR=${LAMBDA_EXECUTOR- }
      - HOST_TMP_FOLDER=${TMPDIR:-/tmp/}localstack
      - DOCKER_HOST=unix:///var/run/docker.sock
      - LAMBDA_DOCKER_NETWORK=localstack
    volumes:
      - "${TMPDIR:-/tmp}/localstack:/tmp/localstack"
      - "/var/run/docker.sock:/var/run/docker.sock"
    networks:
      - localstack

networks:
  localstack:
    name: localstack
```



### Environment

```markdown
- OS: Windows 10
- LocalStack: 0.12.12
```


### Anything else?

_No response_

**Comment by dfangl** [comment]
Hi! Do you have some kind of default registry set manually in your docker client?
Can you try pulling from `docker.io/localstack/localstack`?

**Comment by IshakAtLEGO** [comment]
Hi
It turned out that this "error" was not error after all, but e behavior introduced by latest changes in our organisation setting in Docker.
I had this behavior when using Docker Desktop with my company email. It turned out that our Docker Organisation settings dit not allow Community Images (which LocalStack is).
When signed out I was able to use LocalStack image. We have now changed settings and everything works.

Si I am closing this.
